src/stories/Form.stories.js

```diff
@@ -97,6 +98,11 @@ const Template = (args) => {
                 ]}
                 isFormInput
             />
+            <StatePicker
+                inputID="pickState"
+                shouldSaveDraft
+                isFormInput
+            />
```

Comment:
Please add a top margin like the other inputs

Reply:
we have not defined a style prop for StatePicker, we need to create containerStyles prop here

Reply:
or you can just wrap it in a View like we do in all calls of `StatePicker`

Reply:
Ohkay